src/test/java/com/jcabi/ssl/maven/plugin/CacertsTest.java

```diff
@@ -56,17 +58,27 @@
      */
     @Test
     public void importsCertificatesFromKeystore() throws Exception {
-        final File keystore = this.temp.newFile("keystore.jks");
-        keystore.delete();
-        final File truststore = this.temp.newFile("cacerts.jks");
-        truststore.delete();
+        final File keystoreFile = this.temp.newFile("keystore.jks");
+        keystoreFile.delete();
+        final File truststoreFile = this.temp.newFile("cacerts.jks");
+        truststoreFile.delete();
         final String pwd = "some-password";
-        new Keystore(pwd).activate(keystore);
-        new Cacerts(truststore).imprt();
+        new Keystore(pwd).activate(keystoreFile);
```

Comment:
I see it as that truststore and cacerts are synonyms: these words mean the same. And having `truststore` to denote a file while `cacerts` denotes the file's representation as java object - is not clear to me.

At the same time I see your article's point of naming the vars as their classes - it's logical.
Just for me, `keystoreFile`, `truststoreFile` and `truststore` is clearer than `keystore`, `truststore` and `cacerts`.

But I'm not going to raise a holly-war here ), so I'll rename.